Header header
int32   frame_height
int32   frame_width
float32 inference_time
int32 	no_of_road_markings
int32[]  polygons
int32[]  class_ids
